### DISCUSSION TITLE
Tab color on activity

### DISCUSSION THREAD

**Discussion by aryklein** [kovidgoyal/kitty#4061] [Ideas]
Is there any way to change the tab color or the tab title color when you have activity, instead of adding a symbol (`tab_activity_symbol`)?

**Comment by kovidgoyal** [comment]
No there isn't. Though in the next release you can write your own python function to draw the tab however you like.

**Reply by aryklein** [reply]
Thanks

**Reply by Gfarian** [reply]
> No there isn't. Though in the next release you can write your own python function to draw the tab however you like.

Hello @kovidgoyal is now any way to do so? and is this for firefox?
